test/functional/mempool_ephemeral_dust.py

```diff
@@ -47,6 +47,22 @@ def add_output_to_create_multi_result(self, result, output_value=0):
 
         result["new_utxos"].append({"txid": new_txid, "vout": len(result["tx"].vout) - 1, "value": Decimal(output_value) / COIN, "height": 0, "coinbase": False, "confirmations": 0})
 
+    def create_ephemeral_dust_package(self, *, tx_version, dust_tx_fee=0, dust_value=0, num_dust_outputs=1, extra_sponsors=None):
+        """Creates a 1P1C package containing ephemeral dust. By default, the parent transaction
+           is zero-fee and creates a single zero-value dust output, and all of its outputs are
+           spent by the child."""
+        dusty_tx = self.wallet.create_self_transfer_multi(fee_per_output=dust_tx_fee, version=tx_version)
```

Comment:
```
fee_per_output=dust_tx_fee
```

Since technically it's the fee per output, let's call it `dust_tx_fee_per_output`? Otherwise reading the function signatures gives an impression that `dust_tx_fee` is for the whole tx.

Reply:
Oh interesting, thanks for the clarification. It was seeing `fee_per_output=dust_tx_fee` together that threw me off. Maybe an argument comment here at this call site of `create self transfer multi` can be more clarifying stating that there is only 1 output created initially? That is if the file is retouched.